packages/react-debug-tools/src/__tests__/ReactHooksInspection-test.js

```diff
@@ -67,6 +67,53 @@ describe('ReactHooksInspection', () => {
     ]);
   });
 
+  it('regression test: should inspect a simple custom hook with the name useState (#20613)', () => {
+    const effect = () => {};
+    function useState(value) {
+      const [state] = React.useState(value);
+      React.useDebugValue('custom hook label');
+      React.useEffect(effect);
+      return state;
+    }
+    function Foo(props) {
+      const value = useState('hello world');
+      React.useState('hello world 2');
+      return <div>{value}</div>;
+    }
+    const tree = ReactDebugTools.inspectHooks(Foo, {});
+    expect(tree).toEqual([
```

Comment:
I think this is the right tree, but idk. What matters now is that it fails with:

<img width="804" alt="Screen Shot 2021-01-18 at 7 42 21 PM" src="https://user-images.githubusercontent.com/2440089/104981137-0fed1080-59d6-11eb-97cf-929caf5bf865.png">